plugin/src/test/groovy/org/gradle/testretry/testframework/SpockFuncTest.groovy

```diff
@@ -304,25 +302,29 @@ class SpockFuncTest extends AbstractFrameworkFuncTest {
         def result = gradleRunner(gradleVersion).buildAndFail()
 
         then:
-        result.output.count('passingTest PASSED') == (isRerunsParameterizedMethods() ? 1 : 2)
+        result.output.with {
+            assert count('passingTest PASSED') == (isRerunsParameterizedMethods() ? 1 : 2)
 
-        result.output.count('unrolled[0] PASSED') == 2
-        result.output.count('unrolled[1] FAILED') == 2
-        result.output.count('unrolled[2] PASSED') == 2
+            readLines().with {
+                assert count { it =~ /unrolled ?\[.*?0] PASSED/ } == 2
```

Comment:
spock 1 displays `unrolled[0]` while Spock 2 adds the parameters to the name, like `unrolled [p1:v1, p2:v2 #0]`